osu.Game.Rulesets.Osu/UI/Cursor/OsuCursorSprite.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Rulesets.Osu.UI.Cursor
+{
+    public class OsuCursorSprite : CompositeDrawable
+    {
+        public Drawable ExpandTarget { get; protected set; }
```

Comment:
xmldoc

Reply:
I am not very sure how to document this property. Would like to say "The part of a cursor that expands on mouse down" to clarify its primary purpose, but this seems to repeat variable name.

Reply:
Added documentation for you.